src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,85 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, {
+  ChangeEvent, Dispatch, FormEvent, SetStateAction, useState,
+} from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  setMovies: Dispatch<SetStateAction<Movie[]>>,
+};
+
+export const FindMovie: React.FC<Props> = ({ setMovies }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState('');
+  const [clicked, setClicked] = useState(false);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [sameMovie, setSameMovie] = useState(false);
+
+  const handleFindMovie = async (event: FormEvent<HTMLFormElement>) => {
+    event?.preventDefault();
+    setClicked(true);
+    setIsLoading(true);
+    const data = await getMovie(query);
+
+    if (!('Error' in data)) {
+      setError(false);
+
+      const {
+        Plot, Poster, Title, imdbID,
+      } = data;
+
+      // eslint-disable-next-line
```

Comment:
Using `eslint-disable` is a bag practice. Try to avoid it here and in test tasks. Find the way to solve it without `eslint-disable`.